lib/rules/require-fetch-import.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const ERROR_MESSAGE = 'Explicitly import `fetch` instead of using `window.fetch`';
+
+module.exports = {
+  meta: {
+    type: 'problem',
```

Comment:
This seems more like a `suggestion` than a `problem` since nothing is explicitly broken, it's just good practice to use this import.

Reply:
fine by me